webcompat/static/js/lib/bugform.js

```diff
@@ -216,6 +219,60 @@ function BugForm() {
     this.submitButtons.removeClass("is-disabled");
   };
 
+  this.showSimilarBugs = function() {
+    if (!$("body").data("username")) {
+      // Not login
+      return;
+    }
+
+    if (this.suggestDelay) {
+      clearTimeout(this.suggestDelay);
+    }
+    var bugform = this;
+    this.suggestDelay = setTimeout(function() {
+      try {
+        var url = new URL(bugform.urlField.val());
+        if (url.hostname.length > 7) {
```

Comment:
https://github.com/webcompat/webcompat.com/issues/1826#issuecomment-332374279 > "About the search API" 

> "... fire the event starting at the 5th character ..."

Would be great, if we could change the 7 to a 4 then. thanks!

Reply:
@MDTsai Would you like to make the small fix? Or shall I add a follow up? I don't want to block the merge with this small request. :)

Reply:
let's discuss the char length at some other point. Merging, so team can review.